gitstrap.go

```diff
@@ -340,6 +320,19 @@ func (strap *strapCtx) addCollaborators(owner string, repo *github.Repository) e
 	return nil
 }
 
+func getOwner(strap *strapCtx, opt Options) (string, error) {
+	me, _, err := strap.cli.Users.Get(strap.ctx, "")
+	if err != nil {
+		return "", err
+	}
+	org, hasOrg := opt["org"]
+	owner := *me.Login
+	if hasOrg {
```

Comment:
@vassilevsky it seems that we can move it to beginning of the function and skip `Users.Get()` if options has `org` string. (it would be better to implement a unit test first to be sure that the logic didn't change)

Reply:
Totally. Refactored.